install/migrations/update_10.0.x_to_11.0.0/form.php

```diff
@@ -80,9 +83,12 @@
             `description` longtext,
             `forms_categories_id` int {$default_key_sign} NOT NULL DEFAULT '0',
             `usage_count` int unsigned NOT NULL DEFAULT '0',
+            `visibility_strategy` varchar(30) NOT NULL DEFAULT '',
+            `conditions` JSON NOT NULL,
```

Comment:
Would be nice to have some kind of prefix here to indicate that this is the visibility of the submit button, not of the form.

Same for any others variables in others files.

Reply:
If this is mandatory because of some shared code, you can add a method to the interface to define the fields names.